regression/esbmc-solidity/break_4/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+contract.solast
+--sol contract.sol contract.solast --function test_break --incremental-bmc
```

Comment:
Maybe change this to `--contract` only. So that we can also test the global verification.